src/core/js/models/questionModel.js

```diff
@@ -321,7 +321,7 @@ define([
     getResponseType() {}
 
     /** @type {boolean} */
-    get canShowMarking() {
+    get shouldShowMarking() {
```

Comment:
much better!

Reply:
I don't see why not, seeing as it's brand new.